app/models/participant_declaration/ecf.rb

```diff
@@ -3,4 +3,17 @@
 class ParticipantDeclaration::ECF < ParticipantDeclaration
   include RecordDeclarations::ECF
   belongs_to :statement, optional: true, class_name: "Finance::Statement::ECF"
+  validate :validate_backdated_declaration_before_induction_record_end_date
+
+  def validate_backdated_declaration_before_induction_record_end_date
+    return if participant_profile.nil?
+
+    previous_induction_record = participant_profile.relevant_induction_record(lead_provider: cpd_lead_provider.lead_provider)
+
+    return if participant_profile.current_induction_record == previous_induction_record || previous_induction_record.nil?
+
+    if previous_induction_record.end_date < declaration_date
+      errors.add(:declaration_date, I18n.t("declaration_before_end_date"))
```

Comment:
might be bit clearer
```suggestion
      errors.add(:declaration_date, I18n.t("declaration_must_be_before_end_date"))
```

Reply:
👌 Now updated